### PR TITLE
Create operating profit or loss section of P&L

### PR DESCRIPTION
**Pull request by Jamie87**
Creation of Operating Profit Or Loss section of Profit and Loss.
Includes addition to HTML page and P&L transformer.
Resolves:
SFA-1944 Company accounts Web

### CODE REVIEW

**Comment by divya051988** [comment]
Please have a read at the https://github.com/companieshouse/styleguides/blob/master/git/pull-requests.md

File: src/main/java/uk/gov/companieshouse/web/accounts/transformer/profitandloss/impl/OperatingProfitAndLossTransformer.java [hunk 1]
```diff
@@ -0,0 +1,206 @@
+package uk.gov.companieshouse.web.accounts.transformer.profitandloss.impl;
+
+import org.springframework.boot.autoconfigure.kafka.KafkaProperties;
```
Comment (divya051988) [inline src/main/java/uk/gov/companieshouse/web/accounts/transformer/profitandloss/impl/OperatingProfitAndLossTransformer.java]: Please remove Unused imports .

File: src/main/java/uk/gov/companieshouse/web/accounts/transformer/profitandloss/impl/OperatingProfitAndLossTransformer.java [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+import uk.gov.companieshouse.api.model.accounts.profitandloss.GrossProfitOrLoss;
+import uk.gov.companieshouse.api.model.accounts.profitandloss.ProfitAndLossApi;
+import uk.gov.companieshouse.web.accounts.model.profitandloss.ProfitAndLoss;
+import uk.gov.companieshouse.web.accounts.model.profitandloss.operatingprofitorloss.OperatingProfitOrLoss;
+import uk.gov.companieshouse.web.accounts.model.profitandloss.operatingprofitorloss.items.AdministrativeExpenses;
+import uk.gov.companieshouse.web.accounts.model.profitandloss.operatingprofitorloss.items.DistributionCosts;
+import uk.gov.companieshouse.web.accounts.model.profitandloss.operatingprofitorloss.items.OperatingTotal;
+import uk.gov.companieshouse.web.accounts.model.profitandloss.operatingprofitorloss.items.OtherOperatingIncome;
+
+import javax.print.attribute.standard.MediaSize;
```
Comment (divya051988) [inline src/main/java/uk/gov/companieshouse/web/accounts/transformer/profitandloss/impl/OperatingProfitAndLossTransformer.java]: Same as above.

File: src/main/java/uk/gov/companieshouse/web/accounts/transformer/profitandloss/impl/OperatingProfitAndLossTransformer.java [hunk 3]
```diff
@@ -0,0 +1,206 @@
+package uk.gov.companieshouse.web.accounts.transformer.profitandloss.impl;
+
+import org.springframework.boot.autoconfigure.kafka.KafkaProperties;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.model.accounts.profitandloss.GrossProfitOrLoss;
```
Comment (divya051988) [inline src/main/java/uk/gov/companieshouse/web/accounts/transformer/profitandloss/impl/OperatingProfitAndLossTransformer.java]: Same as abobe

File: src/test/java/uk/gov/companieshouse/web/accounts/transformer/profitandloss/OperatingProfitAndLossTransformerTests.java [hunk 4]
```diff
@@ -0,0 +1,280 @@
+package uk.gov.companieshouse.web.accounts.transformer.profitandloss;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.autoconfigure.kafka.KafkaProperties;
```
Comment (divya051988) [inline src/test/java/uk/gov/companieshouse/web/accounts/transformer/profitandloss/OperatingProfitAndLossTransformerTests.java]: Unused imports

File: src/test/java/uk/gov/companieshouse/web/accounts/transformer/profitandloss/OperatingProfitAndLossTransformerTests.java [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+
+    private OperatingProfitAndLossTransformer transformer = new OperatingProfitAndLossTransformer();
+
+    @Test
+    @DisplayName("Add current period to web model")
+    void addCurrentPeriodToWebModel() {
+
+        ProfitAndLossApi currentPeriodProfitAndLossApi = new ProfitAndLossApi();
+
+        GrossProfitOrLoss grossProfitOrLoss = new GrossProfitOrLoss();
```
Comment (divya051988) [inline src/test/java/uk/gov/companieshouse/web/accounts/transformer/profitandloss/OperatingProfitAndLossTransformerTests.java]: grossProfitOrLoss not required

File: src/test/java/uk/gov/companieshouse/web/accounts/transformer/profitandloss/OperatingProfitAndLossTransformerTests.java [hunk 6]
```diff
@@ -0,0 +1,280 @@
...
+import uk.gov.companieshouse.web.accounts.transformer.profitandloss.impl.GrossProfitAndLossTransformer;
+import uk.gov.companieshouse.web.accounts.transformer.profitandloss.impl.OperatingProfitAndLossTransformer;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+public class OperatingProfitAndLossTransformerTests {
+
+    private static final Long CURRENT_GROSS_TOTAL = 1L;
```
Comment (divya051988) [inline src/test/java/uk/gov/companieshouse/web/accounts/transformer/profitandloss/OperatingProfitAndLossTransformerTests.java]: Unused field .

Comment (Jamie87) [reply src/test/java/uk/gov/companieshouse/web/accounts/transformer/profitandloss/OperatingProfitAndLossTransformerTests.java]: Done

**Pull request merged**